src/wazuh_testing/modules/modulesd/configuration.py

```diff
@@ -0,0 +1,6 @@
+# Copyright (C) 2015-2023, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+# Internal configuration options
+LOCAL_INTERNAL_OPTIONS = 'wazuh_modules.debug'
```

Comment:
Maybe a more specific name?
```suggestion
MODULESD_DEBUG = 'wazuh_modules.debug'
```

Reply:
Fixed with https://github.com/wazuh/qa-integration-framework/pull/11/commits/cab526fee44da8631084c580c3c5ca79a91ea38a